CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.3.0
+
+TBD
```

Comment:
Please add there at least something.

Reply:
we decided to do a quick RC release so colin is usable; we will follow with full release soon-ish